### PR TITLE
feat: add aws ancillary resources

### PR DESCRIPTION
**Pull request by dibyadhar**
#### 📲 What

Creates supporting backend in AWS to store terraform state file remotely

#### 🤔 Why

To prevent local storage of state file and race around conditions if module is executed parallelly

#### 🛠 How

[Configure backends in AWS ](https://www.terraform.io/language/settings/backends/s3)

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.

#### 🕵️ How to test

Run terraform apply by setting up correct input values follow the [example directory](https://github.com/amido/stacks-ancillary-resources/tree/feature/aws/src/aws/backend_infra/example)  

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

**Comment by russellseymour** [comment]
I do not think that we need to sub divide the infra for AWS into another folder as it is only for TF Storage

**Comment by dibyadhar** [comment]
> I do not think that we need to sub divide the infra for AWS into another folder as it is only for TF Storage

Hi @russellseymour updated the dir structure now

**Review by russellseymour — APPROVED** [review]
Great thanks 👍

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,13 @@
...
+
+# OS generated files #
+######################
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
```
Comment (ElvenSpellmaker) [inline .gitignore:14]: Missing newline

**Pull request merged**